packages/backend/src/scheduler.ts

```diff
@@ -58,7 +70,7 @@ async function onHealthCheck() {
 }
 
 createTerminus(server, {
-    signal: 'SIGINT',
+    signals: ['SIGUSR2', 'SIGTERM', 'SIGINT', 'SIGHUP', 'SIGABRT'],
```

Comment:
graphile was listening to all these signals 

(in fact ignoring SIGTERM before meant that the worker was never gracefully shutting down in kubernetes, e.g. when we upgrade lightdash cloud, the existing containers were getting SIGTERM and we cancelling existing tasks half way through their jobs)